src/lib.rs

```diff
@@ -1,13 +1,47 @@
 use bitcoin::Amount as BitcoinAmount;
+use bitcoin::FeeRate as BitcoinFeeRate;
 use bitcoin::ScriptBuf as BitcoinScriptBuf;
 
+use error::FeeRateError;
 use error::ParseAmountError;
 
 #[macro_use]
 mod macros;
 pub mod error;
 pub use bitcoin::Network;
 
+#[derive(Clone, Debug)]
+pub struct FeeRate(pub(crate) BitcoinFeeRate);
```

Comment:
I don't know if I understand the advantage in making this `crate` visible.

Reply:
totally, will remove `(crate)`